model_compiler/python/caffe_to_nnir.py

```diff
@@ -854,7 +854,7 @@ def extractCaffeNodeInfo(net_parameter, graph, inputsInfo, verbose):
 
 # convert caffe graph to ir graph.
 def caffe_graph_to_ir_graph(net_parameter, input_dims, verbose):
-    graph = IrGraph()
+    graph = IrGraph(False)
```

Comment:
Codacy found an issue: ['IrGraph' may be undefined, or defined from star imports: nnir (F999)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5754497)